matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/MatchFinder.java

```diff
@@ -35,38 +36,25 @@
      */
     int getPriority();
 
-    /**
-     * Returns a name which can be used to select a specific matcher to be run.
-     * @return a name
-     */
-    String getName();
```

Comment:
We should have a `getSupportedServerIdList()` or something like that instead, otherwise there is no way to know which matcher can find matches for which server

Reply:
In fact maybe we want each matcher to register itself as a matcher handling requests to serverIDs X, Y and Z